polling_stations/apps/councils/management/commands/import_councils.py

```diff
@@ -1,13 +1,66 @@
 import html
 import json
+from json.decoder import JSONDecodeError
 import requests
 from django.apps import apps
 from django.contrib.gis.geos import GEOSGeometry, MultiPolygon, Polygon
 from django.conf import settings
 from django.core.management.base import BaseCommand
+from requests.exceptions import HTTPError
+from retry import retry
 from councils.models import Council
 
 
+def union_areas(a1, a2):
+    if not a1:
+        return a2
+    return MultiPolygon(a1.union(a2))
+
+
+TEMP_CONTACT_DETAILS = {
```

Comment:
Hopefully we'll be able to fill some more of this stuff in fairly soon. I'll follow up with the Electoral Commission again when we meet them next month.